docs/getting-started.rst

```diff
@@ -79,6 +87,14 @@ displayed using this command::
                           Set amount of displayed information
     -s, --stop-on-failure
                           Stop running checks after a failed one.
+    --map-charm MAP_CHARM
+                          WARNING: This option can lead to failed verifications
+                          when used incorrectly. This option allows user to
+                          explicitly specify which charm is the application
+                          running. It is useful when charm is deployed from
```

Comment:
If the suggestion above is followed, I'd suggest: "Typical use cases involve the usage of local charms or non-official charmhub repositories"